examples/RM3NewPTOSim/RM3Files/README.md

```diff
@@ -0,0 +1,11 @@
+# PTO-Sim Compressible Hydraulic PTO
+
+**Author:**  	Ratanak So
+
+**Version:** 	WEC-Sim v2.0
```

Comment:
@jleonqu You will need to update the WEC-Sim version number that this updated example will be compatible with.